src/utils/db.ts

```diff
@@ -31,7 +31,7 @@ export const createGoal = async (activityId: string, input: GoalCommon) => {
             ${activityId},
             ${input.state.id},
             ${input.priority},
-            max("scopeId") + 1
+            count(*) + 1
```

Comment:
its counting all rows in goals table, really we're needs this?

Reply:
No all in table, all where projectId === selected project id

Reply:
`FROM "Goal" WHERE "projectId" = ${input.parent.id};`